src/schema/cripto.schema.ts

```diff
@@ -0,0 +1,19 @@
kl2z2DqUW8gbwxNWFV/fnS7eaEwPQRMHXLN4pnZnoFXLhM5Vl2KB+Gdos3hq9mu8SkOEvldkSZ9QHxRvDf/WC6TuVJKBW+6uiMuZ9mm2VhvspRz/OmeOhZhIhprsr+0gvdLx52Vs16AKXiVSqLsks8wbpPPOTMm3rtk63K4DjPDvlwNVDpHtm1+F2Tou2wSFbGidCqCvFaF+lQ8wSQiS9nazMxYg2jHxQPM48Amdi/VD7LTv4Nzd9IB5lPgDheV/Ql5e19j4uz6nkCyAyNVq8uXqR8kl0A5ex6nrwBZOKbauPVWut4U590aAEgHJdBpG9TPghNarLTE6R57IGMNfUimvRu5brA9Ef5sAJAGC4Dpt8r56cqJFcUnk4PUWs3nSf/n+Rktm3j6pah6i4K4X1b0vBUhryPgc6fAm0zXIRorXJRKNEXxMORBqcyjPueAzbdAJh6eXiOOtLXKtcU7/GWDkYalsqpQHoWaP24PDyxqUm5JBj9KflQHAeo0g0MdgBbb5fPybpUvEa5IjCtFg/EB4AqQ95+r3xWhv6lHFok5kAgHgFU6QZe/I58XlHddBcmTO3hTvJh+AbQYAfQNl02L6ZHZOGJVNGpk8GkoagkdO8zgCkKAqT32KMB5QAyhfni0IwFoNH8jYbJQON7gdZpQPvNwM4Hgf6No4do42eEc6Mb5QBi/ffcATScy+8Etv0AOPhy8jL89zHZzFg/LQNP48RznNvbJQWQGfLGz1XvbZKU4eCLQPlMSYAOiUyyOrlkQxogo4Gunycvx/qrpLP7lgRAfxYHxkWAWj6QvBxbbpXObln6E6eGXhUHxkWAjK5OpiYpx/ZFsrmx3m93z6QHUFSYKg6LywD1LElWjh2/kM8uzWUeQy8uka+0qwC98lCycuz6rXx2TPekB1D81hsHgHEVoFf/lKwcXT+Tz87o5SkCRF0OVNhdgPb8M1k5tt0tnx1TR3ova3P1IEwXAGq+CKh0JSvHC3MdWe6awkvz4rfzSVfWZYC6H0hYjs7q1Ih0dtZRYWrtAeLCzeIVdRWg9puAwe3JyvDaUvnc0pwXg6GfyFfUMYAazgY670sOj/WmL8vndtRMC2sPUPyC2hwDZOetmi6ozsZv/jqw6yFgYEvyv23du0J2OccJpidqD5ChKDMASfrgS1UIpTM71kyrUgBIvyBeUe8B6gLab5HP60RvSAEg2uVARf0FaHA7sOUb8lkNZ6YdKQCk94lX1FeA9jcBG66Rz+mkAOk9tQeI9UHxivoGUP/z1dFmuxxWOqNTAzRQe4Di9587UFmfAOp9prrmp/Fc+YxO7cMBIJc9sBno+KHLPVEKAIVL2NhB2lcC1l0uDYvYJSzcRI9Lb7QFaLsihzfRWXqMH8vfajgLWHspsOkrwK7fAYPbkv99u5/MfkauHuNZbxSvqAsAvd0tFwM9T47iBvtvLp0dtD4/J3G4BpCxR6ZMAHb+Jnk5nBlYTOHEjtxPpo4I0cTqY3uim2p2ZFI1ncnURfIVdbQHMkNum5m8LOtny+dn6Me1B4hpnnxFHQfIaKAvYS/U8SP5/JhuqD1ApcLHxCvqA0Dbvu/fysRyYUrtAQqL6k+vMTbMTgbQ/sZ8LKqPIQrbekZukOb3JwOo/z/52NZTBUgvE+9uXb+ENUxOvk4oNxsLw9bmkRukNCn5RKskQEx3pwdQuTBFvLdxvQdq+WCyshz4t3B+dTPTAygc7zJyg7Tfkgyg3qfzc7xLDBHrpeI9jss9UN9z/pzQwfovqcITAxTpO8SBcRWgrd9MXpYtCySzuz19gMIhmxi2MTbNS37QpnXLJVK9z5tibzIE6xXivY4rPVDLJdV1QUnL8NaB40KZsf6rCDxOHrSQJkB2gXzrNGDrbUDPE6PbF3/UttfK8vzXSQEKrzrAmL13dXUdUR5fdRBDZOhRb3sgcXcC62cJZkYPi8ITA8R1M8TB8RWgl++UzatcmKZckD3VQRwe3wDqvE82K6Z/KFcE1p8Uh8cXgAY7gJe+K58VF2crlwRDZfFQpOGojGB7eqtd7iqdE1Ojck1guk48GGlAKifx3jXA5q/JPW2d6M8q11SdYNXrAkDdVWjeaAN2/GroOBcXdlwcNTUrVxU/kUmeIS3d01S6gfb5bu02Pda2bUr1VyiXBabHxIMaK2S1+F3jgh0Y9xlJaFKTwfS6fFgBIBwPTy9Wnnm28kH2DXjioASA8LYMblW+CFBnwFBJHJbhnMtLGEX2IUf5JET1F8DQbvnwcg4Q6z6UixcqHxWPDbE+Ih5iXgFifQRMX1Q+C6x/LR5kfgF6QPkuRKoOTA3iYeYOIGpOfadFrYRV9eeDqVs+1LwARF1oqD9PZUlg+hCYXpMPN+um3YjoUpVFDS0+e0M+5Iya9QBMcZbKslDSn4bRFfGws2bWhxDpa1UeBFOYH1dYOvSsmG2WhfkqT4p7ItYHxMP33/1gmqvyqOryD4cnXl03677M3/OMJPvE4MwjvlemLjTQh6Xbz6Vxokb5RvHETA2ZG+cZlxFro+8XXdHoujmeV1ycmRHmWgiRngNDPeKN5Zxpt52clm4ff5aCMBn5RnPFFNnLvHS7eKV4lwcXF+R6+oN1H5jusgv0pNvDWyFS74qv+y6/q9WMOzhHYGiJ2MFPWZTdjgLWa8Ubt+amZjs+Jp13ZoWoeLUzhzmY8ex1qAFM13u3dtlvkPQK55bMmlGAE+k50nnmVojqpoPpkfi0La+mIOhhZ87nCVLKHtUWXwKMfio+eVQaEnOCD8PQyvjJslVNCG3msOzTy9A51sviU9jloNlnD/GON1r6sis06HjZYX9w8RNgusceaWtfa1Sj6ZLD8Wcbvdy+uARR/ZV2eia0R2Yvd4XLUKKbwLQQTI/HT3asn4ehnfH91LFrlezX8fdoB4zeUP3Z+HcWwtCNiApTxU87DQoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKUuOp/wMmNkn83ycv0gAAAABJRU5ErkJggg==',
```

Comment:
Why not to use image files?

Reply:
I don't now. Wanted to try another file type. I'm new in react-native, and wanted to check another shape. I modify it?

Reply:
in feature 2 i'm using images of api.